src/SessionSetup/OrbitServiceInstanceTest.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock-matchers.h>
```

Comment:
Nit: gmock/gmock.h is the public header we should include

Reply:
done